src/handlers.rs

```diff
@@ -655,13 +623,10 @@ mod test {
             key: &[127, 0, 0, 1],
         };
 
-        let expected = serialize_host(
-            &test_logger(),
-            Ok(Some(Host {
+        let expected = (Host {
                 addresses: vec![IpAddr::from(Ipv4Addr::new(127, 0, 0, 1))],
                 hostname: "localhost".to_string(),
-            })),
-        );
+        }).serialize().expect("must succeed");
```

Comment:
Style: I'd call the expect case "must serialize". "must succeed" doesn't provide much more info on panicking than a plain `unwrap()`, but `"must serialize" gives a bit more of a better hint what went wrong if you're debugging tests.